### PR TITLE
fix(client): If no guzzle client injected create a new instance.

### PR DESCRIPTION
**Pull request by NeilMasters**
Currently the oembed system is dieing because no client is ever injected.

### CODE REVIEW

File: Client/GuzzleClient.php [hunk 1]
```diff
@@ -3,6 +3,7 @@
 namespace Markup\OEmbedBundle\Client;
 
 use GuzzleHttp\Exception\GuzzleException;
+use \GuzzleHttp\Client as BaseGuzzleClient;
```
Comment (shieldo) [inline Client/GuzzleClient.php]: Doesn't need backslash at the start, and `Base` as a prefix is a bit confusing.

**Comment by shieldo** [comment]
Would also create a Guzzle client instance with 5s timeout options set too as Guzzle doesn't set a timeout by default.

File: Client/GuzzleClient.php (hunk 1 above)
Comment (NeilMasters) [reply Client/GuzzleClient.php]: @shieldo what would you like it called then?

Comment (shieldo) [reply Client/GuzzleClient.php]: `Guzzle`?

**Comment by shieldo** [comment]
(This is a really good spot btw and I'm sorry for introducing this!)

**Comment by NeilMasters** [comment]
Amendments made.

**Comment by NeilMasters** [comment]
@shieldo will also follow up with a second PR if we can agree a sensible logging level. This has been an issue for some time but went unnoticed because the logs are just warnings(). Would upping the levels of severity be sensible in your opinion?

**Comment by shieldo** [comment]
@NeilMasters There doesn't seem to be any logging within this bundle, unless I'm misreading things?

**Pull request merged**